esp/public/media/theme_editor/less/variables.less

```diff
@@ -240,6 +241,18 @@
 @navbarInverseBrandColor:                @navbarInverseLinkColor;
 
 
+// Sidebar
```

Comment:
Are none of these used for any of the themes? Is that why they are now commented out?

Reply:
lines 4-8 of this file have the note
 "// Note: Variables that are customized by theme editor
//       have been commented out.  The default values
//       are replicated in variables_custom.less and the
//       selected values will be included in the final
//       LESS output that is compiled to CSS."
so I commented these out since they are customizable